science/hamlib/Portfile

```diff
@@ -1,25 +1,28 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        hamlib hamlib 3.3
+github.tarball_from releases
 
 name                hamlib
-version             3.0.1
 categories          science
 platforms           darwin
-maintainers         nomaintainer
+license             LGPL/GPL
```

Comment:
The `license` does not look valid to me. Do you mean LGPL or GPL? If so, use a space instead of a slash. See https://trac.macports.org/wiki/PortfileRecipes#license.

Reply:
done. `port lint` should check this inconsistency or am I wrong?

Reply:
No. `port lint` does not check the validity of the license at all.

Reply:
sorry I mean, why don't add a license check on `port lint` like others package manager? is there a technical issue?

Reply:
I'm not sure what to tell you except that because no one has done it yet.

Reply:
A ok.